### PR TITLE
remove option to enforce https

### LINKED ISSUE #819: Remove Option To Enforce HTTPS In PufferPanel
As per Discord conversation with Lord_Ralex:

Remove the option to enforce HTTPS in PufferPanel and instead allow the user to set what their desired/designated URL should be (http:// | https://) in the URL itself.

### PR DESCRIPTION
**Pull request by nepcore**
this fixes #819 
i also removed upgrade.sql as it's an exact copy of update.sql and only update.sql is referenced anywhere

### CODE REVIEW

**Comment by LordRalex** [comment]
Update and upgrade were used for 2 different points. 

The idea behind update was it was used for minor releases, such as small database queries. Upgrade was used when it went to a major release. Any changes for the database to go from 1.1.x to 1.2.4 was in the upgrade. From 1.2.0 to 1.2.4 was in update.

You would see that change in a past release, as I believe our docs say you have to go to 1.2.0 first before going to 1.2.4. 

I'm fine removing it right now, just giving some history on why we had the 2 files.

**Comment by LordRalex** [comment]
Please handle the reviews that I had included in your PR so that I can accept this.

**Comment by nepcore** [comment]
Sorry, kinda forgot about this

Ok, having the 2 files for different purposes is fine, but when one's not used anymore it should be deleted to not confuse people imo

File: install/install.php [hunk 1]
```diff
@@ -96,15 +96,13 @@
...
-				('https', 0),
 				('allow_subusers', 0)");
 
-    $params['siteUrl'] = preg_replace("(^https?:\/\/)", "", $params['siteUrl']);
 	$query->execute(array(
 		':cname' => $params['companyName'],
-		':murl' => 'http://' . $params['siteUrl'] . '/',
-		':mwebsite' => 'http://' . $params['siteUrl'] . '/',
-		':aurl' => 'http://' . $params['siteUrl'] . '/assets/'
+		':murl' => $params['siteUrl'] . '/',
```
[the remark this answers is not in the document]
Comment (nepcore) [reply install/install.php]: yeah, makes sense, don't know why i changed that...
especially regarding that the install script strips the protocol anyways...

File: install/update.sql [hunk 2]
```diff
@@ -28,4 +28,7 @@ WHERE scopes NOT LIKE '%server.edit%' AND users.root_admin = 1;
 
 ALTER TABLE users MODIFY `session_id` char(40);
 
+UPDATE acp_settings SET setting_val = IF((SELECT setting_val FROM acp_settings WHERE setting_ref = 'https') = '1', IF(setting_val LIKE 'http://%', CONCAT('https://', SUBSTRING(setting_val, 8)), setting_val), setting_val) WHERE setting_ref = 'master_url';
```
[the remark this answers is not in the document]
Comment (nepcore) [reply install/update.sql]: the only thing i have to say here is... why the heck didn't i do that from the start?
either way, i will fix that (obviously)

**Pull request merged**